### DISCUSSION TITLE
How do I represent secretion reaction? How can I see the references on PathVisio?

### DISCUSSION THREAD

**Discussion by JuliaBatUM** [wikipathways/wikipathways-help#58] [Q&A] [answered]
Hi!
I am currently curating metabolic reactions, and some of the metabolites are excreted into the urine. Is there a way to represent these reactions (.e.g by a translocation reaction) ? We do not want to represent these reactions as being transported out of the cell (meaning we do not want the reaction being represented by an arrow going from the inside to the outside of the cell). If that is not possible, could we use basic interactions with the data node "Path"? or do you have any other suggestion?

Also, I was asking myself if there is a way to see all the references at once that I added to PathVisio?

I hope everything is clear. Thank you very much for your help,
Julia

PS:  Is there any documentation I haven´t seen yet that answers my questions?

**Comment by khanspers** [comment, marked as answer]
Hi @JuliaBatUM,

As of right now, our recommendation for depicting molecular transport is to show two copies of the metabolite in the two relevant compartments, connected by a basic arrow. If the process is aided by membrane proteins/transporters, these should be added with a catalysis reaction pointing to the basic arrow. This depiction is described here: https://academy.wikipathways.org/stages/draw-transport/
The next version of PathVisio will support a specific interaction type for transport. 

Here are a couple of examples: 
https://www.wikipathways.org/pathways/WP4233.html
https://www.wikipathways.org/pathways/WP5396.html

About references, as of right now there is no way to see a full list in PathVisio. It's possible this is planned for PathVisio 4, @mkutmon can you comment?

Most of our documentation on how to model pathways is at [WikiPathways Academy](https://academy.wikipathways.org/path.html), and PathVisio documentation is available on the [PathVisio website](https://pathvisio.org/documentation), see PathVisio Book. We are always interested in feedback on our documentation!

**Reply by JuliaBatUM** [reply]
Hi @khanspers,
Thanks a lot for your help!
Kind regards,
Julia
